README.md

```diff
@@ -39,7 +39,7 @@ and the user rating for the hotel.(Van Rossum and Drake 2009)
 
 ## Report
 
-The final report can be found [here]()
+The final report can be found [here](docs/Vegas_strip_data_report.md)
```

Comment:
Should this not be a proper link instead of a file path?

Reply:
tested it works fine; I think we can leave it as path